absl/synchronization/blocking_counter.h

```diff
@@ -21,6 +21,7 @@
 #define ABSL_SYNCHRONIZATION_BLOCKING_COUNTER_H_
 
 #include <atomic>
+#include <limits>
```

Comment:
Can you share the compiler error message? At first glance I do not see any symbol from `<limits>` that is used in this file.

Reply:
[spack-build-out.txt](https://github.com/abseil/abseil-cpp/files/6670624/spack-build-out.txt)


![image](https://user-images.githubusercontent.com/13285627/122407802-856a0680-cf9f-11eb-9486-c977f93ebb4b.png)